lib/python/treadmill_aws/cli/admin/aws/instance.py

```diff
@@ -129,16 +129,30 @@ def configure(instance):
         required=False,
         help='Request a spot instance',
     )
+    @click.option(
+        '--domain',
+        help='domain name to use instead of default'
+    )
+    @click.option(
+        '--ipa-enroll/--no-ipa-enroll',
+        is_flag=True,
+        default=False,
+        help='enroll host in IPA',
+    )
     @treadmill_aws.cli.admin.aws.ON_AWS_EXCEPTIONS
     def create(
             image, image_account, count, disk_size, key, role, ip_address, eni,
             spot, secgroup, size, subnet, data, instance_profile,
-            hostgroup, hostname):
+            hostgroup, hostname, domain, ipa_enroll):
         """Create instance(s)"""
-        ipa_client = awscontext.GLOBAL.ipaclient
+        if ipa_enroll:
```

Comment:
i do not see where ipa_client being used?

Reply:
further down in call hostmanager.host_create()